Tests/libhostmgrTests/resources/dotenv-fixtures.env

```diff
@@ -0,0 +1,26 @@
+UNQUOTED_STRING=foo
+QUOTED_STRING="foo"
+
+UNQUOTED_STRING_WITH_LEADING_SPACE_IN_VALUE= foo
+UNQUOTED_STRING_WITH_TRAILING_SPACE_IN_VALUE=foo 
+
+STRING_WITH_QUOTED_LEADING_SPACE=" foo"
+STRING_WITH_QUOTED_TRAILING_SPACE="foo "
+
+STRING_WITH_LEADING_QUOTE=""foo"
+UNQUOTED_STRING_WITH_EMBEDDED_QUOTE=foo"bar
+STRING_WITH_TRAILING_QUOTE="foo""
+STRING_WITH_EMOJI="🛠 Build for Testing"
+
+STRING_WITH_NO_VALUE=
+STRING_WITH_WHITESPACE_VALUE=
+
+UNQUOTED_STRING_WITH_EQUAL_SIGN=foo=bar
+QUOTED_STRING_WITH_EQUAL_SIGN="foo=bar"
+
+IGNORED_STRING_WITH_NO_EQUAL_SIGN
+
+COMMENTED_QUOTED_STRING="foo" # This is a comment
+COMMENTED_UNQUOTED_STRING=foo # This is a comment
```

Comment:
We also need a case for a line that is exclusively a comment, and especially one with a commented-out key/value pair

```
# This is a lone comment
# DO_NOT_PARSE_THIS="this should not have a value"
```